.github/workflows/ci.yml

```diff
@@ -22,6 +22,11 @@ jobs:
           --health-retries 5
         ports:
           - 6379:6379
+
+      rabbit:
+        image: rabbitmq:3.8.14-alpine
```

Comment:
Maybe some health options could be addes as with redis, vía: `rabbitmq-diagnostics status`

Reply:
Something like the following code:
https://github.com/jpwilliams/remit/blob/master/.github/workflows/pushtest.yml#L9-L14